### PR TITLE
{bio}[GCC/11.3.0] fastml v2.3

### PR DESCRIPTION
**Pull request by lexming**
(created using `eb --new-pr`)

### CODE REVIEW

**Comment by smoors** [comment]
@boegelbot: please test @ generoso

**Comment by lexming** [comment]
@boegelbot please test @ jsc-zen2

File: easybuild/easyconfigs/f/fastml/fastml-2.3-GCC-11.3.0.eb [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+toolchainopts = {'cstd': 'c++03'}
+
+github_account = 'sanger-pathogens'
+source_urls = [GITHUB_SOURCE]
+sources = ['v%(version)s.tar.gz']
+checksums = ['c02901d83ab9457852c67e8e14f41a02d2a0653130ed941b288108ba5272611d']
+
+parallel = 1
+
+prebuildopts = "sed -i 's/-depth-32/-depth-64/g' {programs/Makefile*,libs/phylogeny/Makefile} && "
```
Comment (verdurin) [inline easybuild/easyconfigs/f/fastml/fastml-2.3-GCC-11.3.0.eb:21]: @lexming could you add an explanatory comment about this please?

**Review by verdurin — CHANGES_REQUESTED** [review]

File: easybuild/easyconfigs/f/fastml/fastml-2.3-GCC-11.3.0.eb (hunk 1 above)
Comment (lexming) [reply easybuild/easyconfigs/f/fastml/fastml-2.3-GCC-11.3.0.eb:22]: sure, done in https://github.com/easybuilders/easybuild-easyconfigs/pull/19357/commits/25bdcf006df62129221b56f01cd2a7a2569124f1

**Review by smoors — APPROVED** [review]
lgtm

**Comment by smoors** [comment]
Going in, thanks @lexming!

**Pull request merged**